src/Carbon/Lang/lt.php

```diff
@@ -35,42 +35,56 @@
  * - Justinas (Gamesh)
  */
 return [
-    'year' => ':count metus|:count metus|:count metų',
+    'year' => ':count metai|:count metai|:count metais',
```

Comment:
`':count metai|:count metai|:count metų'` as in 1 metai| 2 metai | 11 metų

Reply:
Updated.